### PR TITLE
feat: add ben and nis

### PR DESCRIPTION
**Pull request by BenjaminMoe**
image

### CODE REVIEW

**Review by nissimsan — APPROVED** [review]

File: docs/spec/index.html [hunk 1]
```diff
@@ -82,6 +82,21 @@
...
+            url: 'https://github.com/nissimsan',
+            company: 'Transmute',
+            companyURL: 'https://www.transmute.industries/',
+          },
+          {
+            name: 'Benjamin Collins',
+            url: 'https://github.com/BenjaminMoe',
+            company: 'Transmute',
+            companyURL: 'https://www.transmute.industries/',
+          },
```
Comment (brownoxford) [inline docs/spec/index.html:97]:
```suggestion
          },
          {
            name: 'Chris Abernethy',
            url: 'https://github.com/brownoxford',
            company: 'mesur.io',
            companyURL: 'https://www.mesur.io/',
          },       
          {
            name: 'Russell Hofvendahl',
            url: 'https://github.com/rhofvendahl',
            company: 'mesur.io',
            companyURL: 'https://www.mesur.io/',
          },              
```

**Review by OR13 — APPROVED** [review]

**Comment by mkhraisha** [comment]
Merge out of band please when ted and chris are added in.

File: docs/spec/index.html (hunk 1 above)
Comment (TallTed) [reply docs/spec/index.html:97]:
```suggestion
          },
          {
            name: 'Chris Abernethy',
            url: 'https://github.com/brownoxford',
            company: 'mesur.io',
            companyURL: 'https://www.mesur.io/',
          },
          {
            name: 'Russell Hofvendahl',
            url: 'https://github.com/rhofvendahl',
            company: 'mesur.io',
            companyURL: 'https://www.mesur.io/',
          },
          {
            name: 'Ted Thibodeau Jr',
            url: 'https://github.com/TallTed',
            company: 'OpenLink Software, Inc',
            companyURL: 'https://www.openlinksw.com/',
          },
```

File: docs/spec/index.html [hunk 2]
```diff
@@ -82,6 +82,33 @@
...
+            url: 'https://github.com/brownoxford',
+            company: 'mesur.io',
+            companyURL: 'https://www.mesur.io/',
+          },       
+          {
+            name: 'Russell Hofvendahl',
+            url: 'https://github.com/rhofvendahl',
+            company: 'mesur.io',
+            companyURL: 'https://www.mesur.io/',
+          },
```
Comment (TallTed) [inline docs/spec/index.html:109]:
```suggestion
          },
          {
            name: 'Ted Thibodeau Jr',
            url: 'https://github.com/TallTed',
            company: 'OpenLink Software, Inc',
            companyURL: 'https://www.openlinksw.com/',
          },
```

**Pull request merged**